zerodb/models/__init__.py

```diff
@@ -1,12 +1,28 @@
 import persistent
 import fields
 import exceptions
+from collective.indexing.queue import getQueue
+from collective.indexing.subscribers import objectModified
+from zope.lifecycleevent.interfaces import IObjectModifiedEvent
+from zope.lifecycleevent import ObjectModifiedEvent
+from zope.lifecycleevent import modified
+from zope.event import subscribers
 from zerodb.intid import IdStore
 from zerodb.trees import family32
 from zerodb.catalog import Catalog
 from zerodb.models.fields import Field, Text
 
 
+def zerodb_autoreindex_dispatcher(event):
+    if isinstance(event, ObjectModifiedEvent):
+        objectAutoReindex(event)
```

Comment:
I guess, the original idea to make a *queue* for re-indexing was to be able to re-index at commit time. E.g. if I change page.title and page.text, I wouldn't want to reindex the object twice